### PR TITLE
TECH-1658: Use JDK 11 image in the workflows

### PR DESCRIPTION
**Pull request by rknj**
## JIRA

https://jira.jahia.org/browse/TECH-1658

## Description

Everything is in the title

### CODE REVIEW

**Review by bpapez — DISMISSED** [review]

**Comment by rknj** [comment]
The version on the master branch is still compatible with Jahia 8.1 which is still compatible with JDK 8 so we cannot update the build to use JDK 11 in this case. So we might need to focus on the run part (Sonar and Tests)

File: .github/workflows/on-code-change.yml [hunk 1]
```diff
@@ -57,4 +57,4 @@ jobs:
           primary_release_branch: master
           github_pr_id: ${{github.event.number}}
           sonar_url: ${{ secrets.SONAR_URL }}
-          sonar_token: ${{ secrets.SONAR_TOKEN }}
\ No newline at end of file
+          sonar_token: ${{ secrets.SONAR_TOKEN }}
```
Comment (bpapez) [inline .github/workflows/on-code-change.yml:60]: Could you please also add the new optional parameter

`nvd_apikey: ${{ secrets.NVD_APIKEY }}`

which we should add on occasion, where it is still missing

Comment (bpapez) [reply .github/workflows/on-code-change.yml:60]: (or it looks like you did not work on the latest version)

Comment (rknj) [reply .github/workflows/on-code-change.yml:60]: I've updated the branch

Comment (bpapez) [reply .github/workflows/on-code-change.yml:60]: thanks

**Review by bpapez — APPROVED** [review]

**Pull request merged**